python/apps/taiga/src/taiga/workflows/services/__init__.py

```diff
@@ -128,7 +129,58 @@ async def update_workflow_status(
     updated_status = await workflows_repositories.update_workflow_status(workflow_status=workflow_status, values=values)
 
     await workflows_events.emit_event_when_workflow_status_is_updated(
-        project=workflow.project, workflow_status=updated_status
+        project=workflow_status.project, workflow_status=updated_status
     )
 
     return updated_status
+
+
+##########################################################
+# delete workflow status
+##########################################################
+
+
+async def delete_workflow_status(workflow_status: WorkflowStatus, move_to_status_slug: str | None) -> None:
+    """
+    This method deletes a workflow status, providing the option to first migrating its stories to another workflow
```

Comment:
is this here and in the API?

Reply:
I don't understand your question.

Reply:
this text is the same as the text in the QueryValidator that you wanted to move to the API (If I understood correctly)

Reply:
Not exactly. This snipped document lives just in the service method, and the text in QueryValidator was for the API doc. I've updated the `api/__init.py__` description to include the `moveTo` param there as a temporal solution until we find the FastApi way of documenting it properly.